client/components/app.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import Header from './header';
+import ProductList from './product-list';
 
 export default class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       message: null,
-      isLoading: true
+      isLoading: false
```

Comment:
For this, you want to leave isLoading as true, that way the page won't try to load until your componentDidMount method finishes running :) At the end of that method, it will set is Loading to false for you.